addons/addons.json

```diff
@@ -173,5 +173,6 @@
   "editor-buttons-reverse-order",
   "columns",
   "editor-compact",
-  "editor-theme3"
+  "editor-theme3",
+  "open-in-gandi"
```

Comment:
New addons are supposed to go above Line 153. Note where it says "NEW ADDONS ABOVE THIS ↑↑."

Reply:
idk how i missed that...